pipeline_svc/conflict_manager.go

```diff
@@ -366,15 +366,30 @@ func (c *ConflictManager) formatTargetDoc(input *crMeta.ConflictParams) *mc.MCRe
 		// TODO: Remove before CC shipping
 		panic(fmt.Sprintf("error '%v' getting target meta", err))
 	}
-
-	bodylen := 0
-	var spec SubdocMutationPathSpec
-	specs := make([]SubdocMutationPathSpec, 0, 6)
 	pv, mv, err := targetMeta.UpdateMetaForSetBack()
 	if err != nil {
 		// TODO: Remove before CC shipping
 		panic("setback unexpected values")
 	}
+	sourceDoc := crMeta.NewSourceDocument(input.Source, input.SourceId)
+	sourceMeta, err := sourceDoc.GetMetadata()
+	if err != nil {
+		// TODO: Remove before CC shipping
+		panic(fmt.Sprintf("error '%v' getting source meta", err))
```

Comment:
Copied and pasted another panic here.
Is the `GetMetadata()` method being used in production for XDCR mobile?
If so, should we fix the panic?

Reply:
This one is only in CCR code path. But I will look at removing the panic later.